src/Tribe/Plugins_List.php

```diff
@@ -0,0 +1,126 @@
+<?php defined( 'WPINC' ) or die;
+
+if (! class_exists( 'Tribe__Plugins_List' ) ) {
+	/**
+	 * A list of Tribe's major plugins. Useful when encouraging users to download one of these.
+	 */
+	class Tribe__Plugins_List {
```

Comment:
I got you. I was under the impression we were trying to make smaller classes, hence why there is 3. They are all strongly related though. If y'all would prefer one class that merged all of these I'm down.